package.json

```
@@ -23,21 +23,22 @@
     "url": "https://github.com/ChenLi0830/dynamoLight"
   },
   "dependencies": {
-    "aws-sdk": "^2.1281.0"
+    "@aws-sdk/client-dynamodb": "^3.241.0",
+    "@aws-sdk/lib-dynamodb": "^3.241.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.2.4",
-    "@types/node": "^18.11.17",
-    "big.js": "^6.2.1",
-    "chai": "^4.3.7",
-    "jest": "^29.3.1",
-    "prettier": "^2.8.1",
-    "serverless": "^3.26.0",
-    "serverless-dynamodb-local": "^0.2.40",
-    "ts-jest": "^29.0.3",
-    "tslint": "^5.20.1",
+    "@types/jest": "^24.0.17",
+    "@types/node": "^12.6.9",
```

```suggestion
    "@types/node": "^18.11.17",
```

Do we want to downgrade Node.js typing to `^12`? Node.js 12 is already EOL, and we should target our Node.js typing data to `16.x` or `18.x`.

---

Upgraded back to v18. I reverted package.json to the working commit, so a bulk a packages were downgraded.